arches/app/views/graph.py

```diff
@@ -562,7 +566,9 @@ def post(self, request, graphid=None):
 
                     clone_data = graph.copy()
                     ret = clone_data["copy"]
-                    ret.slug = None
+                    ret.slug = self.get_slug(
```

Comment:
This should do it, plus a tweak to supply the correct name for "exported_branch" etc
```suggestion
                    ret.slug = Graph.objects.get_slug(
```